CONTRIBUTING.md

```diff
@@ -179,16 +179,57 @@ and instead you should
 ```python
 
 def function_with_many_args(
-    argument1,
-    argument2,
-    argument3,
-    argument4,
-    argument5,
-    argument6,
-    argument7):
+                            argument1,
```

Comment:
When did we start doing this? Maybe I missed it, but looking at the codebase it looks like we don't do this. I.e. https://github.com/raiden-network/raiden/blob/master/raiden/raiden_service.py#L509

Reply:
We do: https://github.com/raiden-network/raiden/blob/master/raiden/api/python.py#L189

And most examples tend to follow the above style. Anything not like this is just gonna need to slowly change. (please don't go making PRs just to change style though :P)

We also had a rocketchat vote on the style and this was decided. Not going back and not discussing this. We got more important things to do.

Reply:
Was just asking for clarification, since I got confused when the first example I found didn't follow this format. Wasn't looking for a discussion!

Reply:
I do however think that what you linked is a coincidence and not on purpose since the rest of that file uses the old format :p

Reply:
As per rocketchat it's 8 spaces -- not align to parentheses. Fixed the example ;)